Homework7/Main.java

```diff
@@ -0,0 +1,15 @@
+package Homework7;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        RunnableThread runnableThread = new RunnableThread();
+        runnableThread.run();
```

Comment:
так стартовать неправильно, у тебя просто вызовется метод run
```suggestion
        Thread runnableThread = new Thread(new RunnableThread());
        runnableThread.start();
```